Dockerfile

```diff
@@ -1 +1,75 @@
+#     Docker hub. More information available
+#     [here](https://arkcase.atlassian.net/wiki/spaces/AANTA/pages/1558446081/Process+for+updating+our+base+image+base+centos).
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/base_centos:7-20210630
+
+LABEL   ORG="Armedia LLC" \
+        APP="Snowbound" \
+        VERSION="1.0" \
+        IMAGE_SOURCE="https://github.com/ArkCase/ark_snowbound" \
+        MAINTAINER="Armedia LLC"
+
+# Environment variables: versions
+ENV SNOWBOUND_VERSION="5.6.2-2021.02.04" \
+    TOMCAT_VERSION="9.0.50" \
+    TOMCAT_MAJOR_VERSION="9"
+
+# Environment variables: Tarball stuff
+ENV SNOWBOUND="VirtualViewerJavaHTML5-$SNOWBOUND_VERSION" \
+    SNOWBOUND_WAR="VirtualViewerJavaHTML5-$SNOWBOUND_VERSION.war" \
+    TOMCAT="apache-tomcat-${TOMCAT_VERSION}" \
+    TOMCAT_TARBALL="apache-tomcat-${TOMCAT_VERSION}.tar.gz" \
+    TOMCAT_TARBALL_SHA512="06cd51abbeebba9385f594ed092bd30e510b6314c90c421f4be5d8bec596c6a177785efc2ce27363813f6822af89fc88a2072d7b051960e5387130faf69c447b" \
+# Environment variables: Java & Tomcat stuff
+    JRE_HOME=/usr/lib/jvm/jre-11-openjdk \
+    CATALINA_HOME=/app/tomcat \
+    CATALINA_PID=/app/tomcat/temp/tomcat.pid \
+    CATALINA_OUT=/dev/stdout \
+    CATALINA_TMPDIR=/app/tomcat/temp \
+    JAVA_FONTS=/app/fonts/.fonts/ \
+# Environment variables: System stuff
+    PATH="/app/tomcat/bin:$PATH"
+
+WORKDIR /app
+COPY "artifacts/${SNOWBOUND_WAR}" \
+    "artifacts/${TOMCAT_TARBALL}" \
+    artifacts/fonts.tar.gz \
+    VirtualViewerJavaHTML5.xml \
+    ./
+
+RUN     set -eu; \
+        checksum=$(sha512sum "$TOMCAT_TARBALL" | awk '{ print $1 }'); \
+        if [ $checksum != $TOMCAT_TARBALL_SHA512 ]; then \
+            echo "Unexpected SHA512 checkum for Tomcat tarball; possible man-in-the-middle attack"; \
+            exit 1; \
+        fi; \
+        yum --assumeyes update; \
+        yum --assumeyes install java-11-openjdk unzip; \
+        yum --assumeyes clean all; \
+        tar xf "$TOMCAT_TARBALL"; \
+        rm "$TOMCAT_TARBALL"; \
+        ln -s "$TOMCAT" tomcat; \
+        rm -rf tomcat/webapps/* tomcat/temp/* tomcat/logs; \
+        useradd --system --user-group --no-create-home --home-dir /app/home tomcat; \
+        mkdir -p home fonts tomcat/webapps/VirtualViewerJavaHTML5 tomcat/conf/Catalina/localhost snowbound-docs; \
+        ln -s /app/snowbound-docs /app/home/.snowbound-docs; \
+        unzip -d tomcat/webapps/VirtualViewerJavaHTML5 "$SNOWBOUND_WAR"; \
+        rm "$SNOWBOUND_WAR"; \
+        chmod 644 VirtualViewerJavaHTML5.xml; \
+        mv VirtualViewerJavaHTML5.xml tomcat/conf/Catalina/localhost/; \
+        tar xf fonts.tar.gz -C fonts; \
+        rm fonts.tar.gz; \
+        cd fonts; \
+        fc-cache -f -v; \
+        cd ..; \
+        chown -R tomcat:tomcat "$TOMCAT" home snowbound-docs; \
+        yum --assumeyes erase unzip; \
+        rpm --erase --nodeps yum; \
+        rm -rf /var/cache/yum
```

Comment:
I am happy with having as little RUN commands as possible (i.e. use multi-line continuations). Using multiple RUN commands can certainly help with debugging, but I think those RUN commands should be collapsed into a single RUN command once the debugging phase is finished. If debugging is needed again, it is easy enough to expand the RUN command as necessary. 
@JonHolman What's your take on this issue?

Reply:
I agree.  Since RUN adds layers, we should try to combine, and with intending and experience we should be used to looking at these.  When debugging, definitely break it up.

Reply:
Well that's the thing - you never know when there will be need to debug and this adds work to that effort. If someone were to leave a continuation in by mistake, or mistakenly whack something b/c of the need to edit, this might add unnecessary effort.

Instead, having it in a debug-friendly model is probably a better idea. Worst case, if there's actually a desire to minimize layers, then an external script might be a cleaner approach, but that would break the concept of keeping as much of the work in the Dockerfile as possible.

I'm fine with whatever you guys choose to do, but I think at this point we're investing effort into something whose returns we're not yet starved for (vs. the ability to quickly edit, modify, etc., as safely as possible).